src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -753,13 +878,50 @@ else if (data.getType() == RmdChunkOutputFinishedEvent.TYPE_INTERACTIVE &&
             // set dirty state if necessary
             setDirtyState();
          }
+
+         if (satelliteChunks_.contains(data.getChunkId()))
+         {
+            forwardEventToSatelliteChunk(
+               docUpdateSentinel_.getId(),
+               data.getChunkId(),
+               new ChunkSatelliteOutputFinishedEvent(
+                  ensureVisible,
+                  data.getScope()
+               )
+            );
+         }
       }
    }
 
+   @Override
+   public void onConsoleWriteOutput(ConsoleWriteOutputEvent event)
```

Comment:
Removed. However, somehow, if I don't manually forward this event, instead of getting single output events to construct the gallery, we get a single event with multiple output where the console is not the first entry. Therefore, the gallery is not constructed at all, unless I make this change:

https://github.com/rstudio/rstudio/pull/765/commits/dcbafa485b7bc082d2fbc249e4465c3e5559da1d